core/commands/bitswap.go

```diff
@@ -28,10 +28,10 @@ var BitswapCmd = &cmds.Command{
 
 var unwantCmd = &cmds.Command{
 	Helptext: cmds.HelpText{
-		Tagline: "Remove a given block from your wantlist",
+		Tagline: "Remove a given block from your wantlist.",
 	},
 	Arguments: []cmds.Argument{
-		cmds.StringArg("key", true, true, "key to remove from your wantlist").EnableStdin(),
+		cmds.StringArg("key", true, true, "Key to remove from your wantlist.").EnableStdin(),
```

Comment:
This is also the case if you grep for `cmds\.stringarg`, e.g. `ipfs log`. It'd be nice to solve this issue once for all.